### DISCUSSION TITLE
Question: Drag and drop using playwright testing libarry

### DISCUSSION THREAD

**Discussion by InduKrish** [testing-library/playwright-testing-library#584] [Q&A]
I have a scenario to test drag and drop ,and validate.

Can you please help how to use Playwright DOM library to test drag and drop? Attached is the screenshot of the screen, 

There is a table with set of rows, and each row is draggable using the dot that appears at the end of each row.
and each row can be dragged and placed to anywhere in the table and see if the row is placed in the table accordingly. and we would like to add the test to verify the same,
Can you please help us with the example or the link to refer to? I searched for the example but couldn't find any.

<img width="1669" alt="Screen Shot 2022-12-27 at 3 55 27 PM" src="https://user-images.githubusercontent.com/113629123/209726013-4638d673-d57e-40ca-9907-5b2f17bdc43b.png">

**Comment by jrolfs** [comment]
https://reflect.run/articles/how-to-test-drag-and-drop-interactions-in-playwright/
